### ISSUE TITLE
Assigning ports to a VLAN on an Aruba 2930M switch 

### ISSUE THREAD

**Issue by aamore** [aruba/aruba-ansible-modules#22]
I'm used to assigning ports on cisco switches. I am new to the HP's switches and commands.
I've gone through the reference guide and googled how to assign ports to certain VLANS and I'm not sure if i'm doing it right.

I'm using these commands:

int vlan 2
tagged 6-24

If anyone has any input or constructive criticism, I'm more than happy to accept! Thanks!

**Comment by dan1eltudares** [comment]
If you want to tag a vlan on an interface, it needs to be done on the physical port or the lag (in case you are doing link aggregation). So for example, if you want to tag Vlan 2 on ports 6 - 24 you can do:
`# vlan 2 tagged 6-24`
If you have a lag interface (for example trk1) them you do: 
`# vlan 2 tagged trk1`
If you want to do an access port just set the vlan as untagged:
`# vlan 2 untagged 6-24`

Hope this helps!

**Comment by aamore** [comment]
This does help. 

So I am trying to have ports 1-5 enabled and traffic able to freely flow through and I used this command:
`# vlan 2 untagged 1-5`

I disabled ports 6-24 and want to put them into a vlan where disabled ports are stored. I used this command:
`# vlan 8 tagged 6-24`

Would this be the correct way to go about it? The information you have given helped, but I'm wondering now if i should have used `# vlan 8 untagged 6-24` instead of `# vlan 8 tagged 6-24`
